### PR TITLE
Change feature flag to RELAYING_MOBILE

### PR DESCRIPTION
**Pull request by biafra23**
Changes proposed in this pull request:
- Allow to disable and enable RELAYING separately for mobile and web.

### CODE REVIEW

**Comment by biafra23** [comment]
Merging this needs to be coordinated with the flag being introduced on staging and production

**Review by MouazAlzahabi — APPROVED** [review]

File: Multisig/Logic/Models/Chain.swift [hunk 1]
```diff
@@ -400,7 +400,7 @@ extension Chain {
         case eip1271 = "EIP1271"
         case eip1559 = "EIP1559"
         case erc721 = "ERC721"
-        case relay = "RELAYING"
+        case relayingMobile = "RELAYING_MOBILE"
```
Comment (fmrsabino) [inline Multisig/Logic/Models/Chain.swift:403]: nitpick: maybe keep the previous name? 🤔 From the mobile perspective we don't need to know that this feature is "mobile specific".

**Review by fmrsabino — APPROVED** [review]
🚀

File: Multisig/Logic/Models/Chain.swift (hunk 1 above)
Comment (biafra23) [reply Multisig/Logic/Models/Chain.swift:403]: I want the enum to have the same name. So if I am looking for it with curl, I am not confused.

**Pull request merged**